job-dsls/jobs/kogito.groovy

```diff
@@ -303,6 +308,60 @@ job("${folderPath}/kogito-cloud-${mainBranch}") {
 
 }
 
+
+// ++++++++++++++++++++++++++++++++++++++++++ Build and deploys kogito-cloud-s2i-images ++++++++++++++++++++++++++++++++++++++++++++++++++
+
+// definition of kogito-cloud-s2i-images script
+def kogitoCloudS2i='''#!/bin/bash -e
+
+# clone the kogito-bom repository
+git clone https://github.com/$ghOrgUnit/kogito-cloud.git -b $mainBranch --depth 50
+# build the project
+cd s2i
+make build
+make test'''
+
+
+job("${folderPath}/kogito-cloud-s2i-images-${mainBranch}") {
+    description("build project kogito-cloud-s2i")
+    parameters {
+        stringParam("mainBranch", "${mainBranch}", "Branch to clone. This will be usually set automatically by the parent trigger job. ")
+        stringParam("ghOrgUnit", "${ghOrgUnit}", "Name of organization. This will be usually set automatically by the parent trigger job. ")
+    }
+
+    label('kogito-static')
```

Comment:
Not sure if this node has all tools you need. Do you need this label specifically? Or could you use a dynamically provisioned machine (from an image which already contains osbs etc.) - for osbs, we have a label 'osbs-builder'.